tuf/client/client.go

```diff
@@ -277,17 +276,18 @@ func (c *Client) downloadTimestamp() error {
 			}
 			return err
 		}
-		logrus.Debug("using cached timestamp")
+		logrus.Debug(err.Error())
+		logrus.Warn("Using cached timestamp - this may mean we cannot get the latest version of the repository")
```

Comment:
can we rephrase this?

"Error while downloading remote metadata, using cached timestamp - this might not be the latest version available remotely"

I would avoid the word "Repository" anywhere in notary.